web/controllers/page_controller.ex

```diff
@@ -1,20 +1,22 @@
 defmodule Nermesterts.PageController do
   use Nermesterts.Web, :controller
   alias Nermesterts.Game
+  alias Nermesterts.Phrase
   alias Nermesterts.Player
 
   def index(conn, _params) do
     games = Nermesterts.Repo.all(Game)
-    players = Nermesterts.Repo.all(Player)
-    num_players = length(players)
+    active_players = Nermesterts.Repo.all(from p in Player, where: p.active == true)
```

Comment:
Ideally, I'd like to keep all of the players together and just filter them on the page